packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts

```diff
@@ -814,9 +861,31 @@ export class PaginatedMessage {
 	/**
 	 * Custom prompt message when a user wants to jump to a certain page number.
 	 * @default "What page would you like to jump to?"
+	 * @remark To overwrite this property change it somewhere in a "setup" file, i.e. where you also call `client.login()` for your bot.
```

Comment:
```suggestion
	 * @remark To overwrite this property change it in a "setup" file prior to calling `client.login()` for your bot.
```
Saying "somewhere" and then using a specific location, implying one was necessary to begin with, is hedging.